packages/dashboard/src/components/ConfirmDialog/index.tsx

```diff
@@ -0,0 +1,68 @@
+import { MaskDialog } from '@dimensiondev/maskbook-theme'
+import {
+    DialogContent,
+    DialogActions,
+    Button,
+    dialogActionsClasses,
+    dialogContentClasses,
+    experimentalStyled as styled,
+    buttonClasses,
+} from '@material-ui/core'
+
+const StyledDialogActions = styled(DialogActions)(({ theme }) => ({
+    [`&.${dialogActionsClasses.root}>:not(:first-of-type)`]: {
+        marginLeft: theme.spacing(3),
+    },
+}))
+
+const StyledDialogContent = styled(DialogContent)(() => ({
+    [`&.${dialogContentClasses.root}`]: {
+        paddingLeft: 90,
+        paddingRight: 90,
+        minHeight: 168,
+        display: 'flex',
+        alignItems: 'center',
+    },
+}))
+
+const StyledButton: typeof Button = styled(Button)(() => ({
+    [`&.${buttonClasses.root}`]: {
+        minWidth: 100,
+    },
+})) as any
+
+interface ConfirmDialogProps extends React.PropsWithChildren<{}> {
+    title: string
+    open: boolean
+    cancelText?: React.ReactNode | string
+    confirmText?: React.ReactNode | string
+    confirmDisabled?: boolean
+    onClose(): void
+    onConfirm(): void
+}
+
+export default function ConfirmDialog(props: ConfirmDialogProps) {
+    const {
+        title,
+        open,
+        onClose,
+        onConfirm,
+        children,
+        cancelText = 'Cancel',
+        confirmText = 'Confirm',
```

Comment:
It should be an i18n string

Reply:
will do i18n replacement for all in another PR.